### PR TITLE
Changed the terminology `Best sales` to `Best sellers`

### LINKED ISSUE #19643: Terminology Best sellers/Best sales
#### Describe the bug

There is used different terminology for the same functionality: Best sellers/Best sales

#### Expected behavior

I expect unified terminology, otherwise it is confusing for a business user

#### Steps to Reproduce

1. e-shop FO
2. Scroll down to 'Products' => see link 'Best sales'
3. Click on 'Best sales' => see 'Best sellers'  in the header of new page

**Screenshots**

image

#### Additional information

* PrestaShop version: 1.7.6.5
* PHP version: 7.1.13

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Changed the terminology 'Best sales' to 'Best sellers'
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #19643
| Related PRs       | N/A
| How to test?      |  * Go to the FO<br>* Scroll down to 'Products' => see link 'Best sales' (now it's 'Best sellers')<br />Click on 'Best sales' (now it's 'Best sellers') => see 'Best sellers' in the header of new page
| Possible impacts? | N/A

### CODE REVIEW

**Review by FabienPapet — DISMISSED** [review]

**Review by Hlavtox — DISMISSED** [review]

**Review by PululuK — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by kpodemski — DISMISSED** [review]

**Comment by kpodemski** [comment]
@l-delin you can put Waiting for QA after wording approval :)

File: classes/lang/KeysReference/MetaLang.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 trans('404 error', 'Shop.Navigation');
-trans('Best sales', 'Shop.Navigation');
+trans('Best sellers', 'Shop.Theme.Catalog');
```
Comment (lartist) [inline classes/lang/KeysReference/MetaLang.php]: HI @Progi1984. Why change the catalogue?

Comment (Progi1984) [reply classes/lang/KeysReference/MetaLang.php]: Because the translation `Best sellers` (title of the page Best Sellers) is existing in an other domain.

Comment (lartist) [reply classes/lang/KeysReference/MetaLang.php]: Ok thanks for your response.

**Review by lartist — DISMISSED** [review]

**Comment by l-delin** [comment]
> @l-delin you can put Waiting for QA after wording approval :)

done 👏

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @Progi1984 

I have checked this PR. I found these issues : 

**Issue 1** : `Best selllers` terminology is not updated in the url when clicking on best sellers

image

image

**Issue 2** : Best sellers is not well translated in Frensh : It was OK Without PR

image


**Without PR : PS 8.0.0 final version**

image

Could you please check it ?

Thank You!

**Comment by Progi1984** [comment]
@PrestaShop/product-team For the issue 1,should i change the URL ?

@PrestaShop/wording-managers For the issue 2, should I use a domain `Modules.Bestsellers.Shop` in the Core or the new translation will be translated in `8.1` ?

**Comment by eshraw** [comment]
@Progi1984 yes, let's change the URL -> consistency between the URL and the content is for the best.

**Comment by Progi1984** [comment]
Thanks @eshraw & @HanaRebaiQA  : URL changed

I wait for the Wording Team.

**Comment by lartist** [comment]
> @PrestaShop/product-team For the issue 1,should i change the URL ?

If we change the url, we should think that people use probably bookmarks, this way their bookmarks won't work anymore. I know it's not the priority, I just want to mention it in case we didn't think about.

**Comment by Progi1984** [comment]
@lartist It's known by @eshraw. After this PR, i will need to update the autoupgrade.

**Comment by l-delin** [comment]
> @PrestaShop/product-team For the issue 1,should i change the URL ?
> 
> @PrestaShop/wording-managers For the issue 2, should I use a domain `Modules.Bestsellers.Shop` in the Core or the new translation will be translated in `8.1` ?

Hello, if this string appears only in this module, keep it in  `Modules.Bestsellers.Shop.` If it's found anywhere else, you can use  `Shop.Navigation` :-)

File: classes/lang/KeysReference/MetaLang.php (hunk 1 above)
Comment (Progi1984) [inline classes/lang/KeysReference/MetaLang.php]:
```suggestion
trans('Best sellers', 'Shop.Navigation');
```

**Comment by Progi1984** [comment]
Thanks @l-delin. It's done. Could you add a "Wording :heavy_check_mark:" label if it's ok for you ?

**Comment by l-delin** [comment]
@Progi1984 Done, thank you!

**Review by ghost — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by paulnoelcholot** [comment]
Hello  @Progi1984,

I have tested this PR and it's OK for me, I didnt found regression :tada: 

Thanks you ! 

image

**Review by paulnoelcholot — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**